.travis-tox.ini

```diff
@@ -131,6 +131,10 @@ commands =
     bash -c "if grep --include '*.py' --include '*.rst' --include '*.tex' -rni 'doi:' Bio BioSQL Scripts Doc ; then echo 'Please use https://doi.org/... not the doi: or DOI: style.'; false; fi"
     bash -c "if grep --include '*.py' --include '*.rst' --include '*.tex' -rn 'dx\.doi\.org' Bio BioSQL Tests Scripts Doc ; then echo 'Please use https://doi.org/... not the dx.doi.org style.'; false; fi"
     bash -c "if grep --include '*.py' --include '*.rst' --include '*.tex' -rn 'http://doi\.org' Bio BioSQL Tests Scripts Doc ; then echo 'Please use https://doi.org/... not http://doi.org/...'; false; fi"
+    # Black style check:
+    black --check .
```

Comment:
One option here would be to provide the full diff if black fails to help people spot the non-conforming lines? e.g. something like `! black --check . && black --diff .`

Reply:
I did wonder about ``black --diff .`` instead of ``black --check .`` but why run both?

Now that all the existing code is black compliant, the diff ought to be modest (unless trying to add entire new files which are not black compliant). Might be worthwhile?

Note if running locally with a cache, I understand ``black --diff`` currently has a performance penalty over ``black --check``, but with TravisCI we don't have the benefit of a black cache anyway.

Reply:
Ah yes of course, running `black --diff .` only would be better- I think it might help new contributors especially (if there is no substantial performance loss)

Reply:
Note that ``black --diff .`` does not set an error code if changes are needed.

I later had to change this to ``black --check --diff .`` instead, see c470fa2b4d14d7f67f597550d334061c664f1c90